source/views/streaming/streams/list.js

```diff
@@ -71,8 +71,11 @@ export class StreamListView extends React.PureComponent {
 
   getStreams = async (date: moment = moment.tz(CENTRAL_TZ)) => {
     try {
-      const dateFrom = date.format('YYYY-MM-DD')
-      const dateTo = date.add(1, 'month').format('YYYY-MM-DD')
+      const dateFrom = date.clone().format('YYYY-MM-DD')
```

Comment:
You don't need the clone here - you're not mutating the date, just making a string from it.

Reply:
Not cloning it means the date is mutated because it's passed by reference. Anything that eventually uses it below the first few lines of `getStreams` would have to make sense of a date that is one month in the future. I want to prevent that.

Reply:
Wait, I think you're pointing to line 74. Your comment is fair.

Reply:
I am indeed pointing to L74. 😄

Reply:
Fixed.